### ISSUE TITLE
tflint without any additional arguments doesn't report anything

### ISSUE THREAD

**Issue by varac** [terraform-linters/tflint#985]
I'm new to tflint, so forgive me any obvious mistake, but I can't get it working for me.

Given this `variables.tf` file: 

```hcl
$ cat variables.tf 
variable "test" {}
```

Simply running `tflint` without any additional arguments doesn't report anything:

```
❯ TFLINT_LOG=debug tflint .                                 
16:41:34 config.go:98: [INFO] Load config: .tflint.hcl
16:41:34 config.go:110: [INFO] Default config file is not found. Ignored
16:41:34 config.go:119: [INFO] Load fallback config: /home/varac/.tflint.hcl
16:41:34 config.go:127: [INFO] Fallback config file is not found. Ignored
16:41:34 config.go:129: [INFO] Use default config
16:41:34 option.go:54: [DEBUG] CLI Options
16:41:34 option.go:55: [DEBUG]   Module: false
16:41:34 option.go:56: [DEBUG]   DeepCheck: false
16:41:34 option.go:57: [DEBUG]   Force: false
16:41:34 option.go:58: [DEBUG]   IgnoreModules: map[string]bool{}
16:41:34 option.go:59: [DEBUG]   EnableRules: []string(nil)
16:41:34 option.go:60: [DEBUG]   DisableRules: []string(nil)
16:41:34 option.go:61: [DEBUG]   Only: []string(nil)
16:41:34 option.go:62: [DEBUG]   Varfiles: []string{}
16:41:34 option.go:63: [DEBUG]   Variables: []string{}
16:41:34 loader.go:57: [INFO] Initialize new loader
16:41:34 loader.go:82: [INFO] Load configurations under .
16:41:34 loader.go:90: [INFO] Module inspection is disabled. Building a root module without children...
16:41:34 loader.go:170: [INFO] Load values files
16:41:34 runner.go:52: [INFO] Initialize new runner for root
16:41:34 provider.go:87: [INFO] Prepare rules
16:41:34 provider.go:123: [INFO]   768 rules enabled
```

Only when I enable the `terraform_typed_variables` rule it will complain:

```
❯ TFLINT_LOG=debug tflint --only=terraform_typed_variables .
16:45:35 config.go:98: [INFO] Load config: .tflint.hcl
16:45:35 config.go:110: [INFO] Default config file is not found. Ignored
16:45:35 config.go:119: [INFO] Load fallback config: /home/varac/.tflint.hcl
16:45:35 config.go:127: [INFO] Fallback config file is not found. Ignored
16:45:35 config.go:129: [INFO] Use default config
16:45:35 option.go:54: [DEBUG] CLI Options
16:45:35 option.go:55: [DEBUG]   Module: false
16:45:35 option.go:56: [DEBUG]   DeepCheck: false
16:45:35 option.go:57: [DEBUG]   Force: false
16:45:35 option.go:58: [DEBUG]   IgnoreModules: map[string]bool{}
16:45:35 option.go:59: [DEBUG]   EnableRules: []string(nil)
16:45:35 option.go:60: [DEBUG]   DisableRules: []string(nil)
16:45:35 option.go:61: [DEBUG]   Only: []string{"terraform_typed_variables"}
16:45:35 option.go:62: [DEBUG]   Varfiles: []string{}
16:45:35 option.go:63: [DEBUG]   Variables: []string{}
16:45:35 loader.go:57: [INFO] Initialize new loader
16:45:35 loader.go:82: [INFO] Load configurations under .
16:45:35 loader.go:90: [INFO] Module inspection is disabled. Building a root module without children...
16:45:35 loader.go:170: [INFO] Load values files
16:45:35 runner.go:52: [INFO] Initialize new runner for root
16:45:35 provider.go:87: [INFO] Prepare rules
16:45:35 provider.go:100: [DEBUG] Only mode is enabled. Ignoring default rules
16:45:35 provider.go:107: [DEBUG] `terraform_typed_variables` is enabled
16:45:35 provider.go:123: [INFO]   1 rules enabled
1 issue(s) found:

Warning: `test` variable has no type (terraform_typed_variables)

  on variables.tf line 1:
   1: variable "test" {}

Reference: https://github.com/terraform-linters/tflint/blob/v0.21.0/docs/rules/terraform_typed_variables.md
```

Any ideas what's going on ?

## Version

* TFLint v0.21.0
* Terraform v0.13.5

**Comment by bendrucker** [comment]
All expected behavior. That config passes the default enabled rules. There are lots of AWS rules enabled by default but few Terraform style rules. `variable "foo" {}` is perfectly valid. 

https://github.com/terraform-linters/tflint/tree/master/docs/rules#terraform-rules

All the Terraform rules in TFLint core implement officially recommended practices from the Terraform docs but most style rules are disabled by default.
